homeassistant/components/netatmo/const.py

```diff
@@ -27,17 +29,38 @@
 EVENT_HUMAN = "human"
 EVENT_ANIMAL = "animal"
 EVENT_VEHICLE = "vehicle"
+EVENT_OUTDOOR = "Outdoor"
+EVENT_CONNECTION = "connection"
+EVENT_HUSH = "hush"
+EVENT_SMOKE = "smoke"
+EVENT_TAMPERED = "tampered"
+EVENT_OFF = "off"
+EVENT_ON = "on"
+EVENT_TAG_BIG_MOVE = "tag_big_move"
+EVENT_TAG_SMALL_MOVE = "tag_small_move"
+EVENT_TAG_OPEN = "tag_open"
 
 EVENT_BUS_PERSON = "netatmo_person"
 EVENT_BUS_MOVEMENT = "netatmo_movement"
 EVENT_BUS_HUMAN = "netatmo_human"
 EVENT_BUS_ANIMAL = "netatmo_animal"
 EVENT_BUS_VEHICLE = "netatmo_vehicle"
+EVENT_BUS_OUTDOOR = "netatmo_outdoor"
```

Comment:
We shouldn't create like 1000 netatmo related events. Instead, you should create 1 event `netatmo_event` and give it a type.